shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Chain.hs

```diff
@@ -208,7 +212,7 @@ chainTransition ::
   TransitionRule (CHAIN crypto)
 chainTransition =
   judgmentContext
-    >>= \(TRC (sNow, ChainState nes cs eta0 etaV etaC etaH lab, block@(Block bh _))) -> do
+    >>= \(TRC ((), ChainState nes cs eta0 etaV etaC etaH lab, block@(Block bh _))) -> do
```

Comment:
I wonder if we should still have the slot number check in the CHAIN rule. The CHAIN rule isn't directly called from consensus, but it (should) conform to the spec, which binds together protocol and ledger concerns at this level to be able to express properties about them.

We could, alternatively, remove these slot number/block number checks from the spec as well, or move them into a separate "envelope" section

Reply:
That's totally fine with me. As long as they are not part of the existing `chainChecks` function, which we do call from consensus.

Reply:
if consensus never calls `CHAIN` then perhaps we could just move the slot number/block number checks (ie `checkPrtclSeq`) into the function `chainChecks`, which is only called in `CHAIN` and currently checks a couple things about the block header anyway (that the pv isn't too far, and the the header and the body are not too big). I do think it would seem weird to me if the ledger allowed applying blocks with slots that were not in order... (the counter argument being that the consensus layer is responsible for this property.)

Reply:
> Looks good. We should decide (with @JaredCorduan) what to do about the syncing with the formal spec here.

Once we've settled on the details, I am happy to make a follow up PR to align the pdf.

Reply:
oops, there is a delay on github that I missed, and I see that I just suggested exactly what you don't want :laughing:

Reply:
I like the idea of just calling `checkPrtclSeq` in the `CHAIN` rule before or after `chainChecks`.

Reply:
Ok, I'll add `checkPrtclSeq`.

Reply:
Pushed a commit adding the check.